module2/src/com/epam/arraysofarrays/MatrixWorker.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.arraysofarrays;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+
+public class MatrixWorker {
+    //task 1
+    public static boolean findOddColumnsWhereFirstElementGreaterThanLast(int[][]matrix) {
```

Comment:
Why do you need boolean? If you don't need to return anything, you can make method of type - void
```suggestion
    public static void findOddColumnsWhereFirstElementGreaterThanLast(int[][] matrix) {
```